internal/cmd/root.go

```diff
@@ -65,8 +65,14 @@ func preRunCheck(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	cmdName := cmd.Name()
+	if cmd.Parent() != cmd.Root() {
+		// e.g. `completion` for `buffalo completion bash` instead of `bash`
+		cmdName = cmd.Parent().Name()
```

Comment:
it is possible that some "current command" is not a direct child of root but a grandchild. they should be checked with the name of their parent.